.github/workflows/playwright.yml

```
@@ -0,0 +1,35 @@
+# Consider moving this to run as part of ci.yml
+
+name: Playwright Tests
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+      - "daily-test-sync"
+
+jobs:
+  playwright:
+    name: 'Playwright Tests'
+    runs-on: ubuntu-latest
+    container:
+      # Note: the version of the docker container for playwright must exactly match the version used in package.json.
+      image: mcr.microsoft.com/playwright:v1.35.1-jammy
+    steps:
+        - uses: actions/checkout@v3
+
+        # Steps for Playwright testing with container
+        - uses: actions/setup-node@v3
+          with:
+            node-version: 18
+        - name: Install dependencies
+          run: |
+            npm ci
+            npx playwright install
```

Ah, this also shouldn't install the browsers since they're supposed to be provided by the docker image:
```suggestion
          run: npm ci
```